metaci/cumulusci/models.py

```diff
@@ -11,6 +16,46 @@
 from django.urls import reverse
 from django.utils import timezone
 from django.utils.dateparse import parse_datetime
+from django.conf import settings
+from simple_salesforce import Salesforce as SimpleSalesforce
+from simple_salesforce.exceptions import SalesforceError
+
+
+def jwt_session(url=None, username=None):
+    if url is None:
+        url = 'https://login.salesforce.com'
+
+    payload = {
+        'alg': 'RS256',
+        'iss': settings.SFDX_CLIENT_ID,
+        'sub': username,
+        'aud': url, #jwt aud is NOT mydomain
+        'exp': timegm(datetime.utcnow().utctimetuple()),
+    }
+    encoded_jwt = jwt.encode(
+        payload,
+        settings.SFDX_HUB_KEY,
+        algorithm='RS256',
+    )
+    data = {
+        'grant_type': 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+        'assertion': encoded_jwt,
+    }
+    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+    auth_url = urljoin(url, 'services/oauth2/token')
+    response = requests.post(url=auth_url, data=data, headers=headers)
+    response.raise_for_status()
+
+    return response.json()
+
+def sf_session(jwt):
+    return SimpleSalesforce(
+            instance_url=jwt['instance_url'],
+            session_id=jwt['access_token'],
+            client_id='metaci',
+            version='42.0'
```

Comment:
We should probably use an env var here but I don't see one already so I won't hold up the PR for that.

Reply:
for sure.